### PR TITLE
notification messages

### PR DESCRIPTION
**Pull request by slavik-lvovsky**

### CODE REVIEW

File: backend/src/yeomanui.ts [hunk 1]
```diff
@@ -8,7 +8,13 @@ import { ReplayUtils, ReplayState } from "./replayUtils";
...
 const titleize = require('titleize'); // eslint-disable-line @typescript-eslint/no-var-requires
 const humanizeString = require('humanize-string'); // eslint-disable-line @typescript-eslint/no-var-requires
-import * as defaultImage from "./defaultImage";
+import * as defaultImage from "./images/defaultImage";
+import * as errorVSCode from "./images/vscode/errorMessage";
+import * as infoVSCode from "./images/vscode/infoMessage";
+import * as warnVSCode from "./images/vscode/warnMessage";
+import * as errorTheia from "./images/theia/errorMessage";
+import * as infoTheia from "./images/theia/infoMessage";
+import * as warnTheia from "./images/theia/warnMessage";
```
Comment (tomer-epstein) [inline backend/src/yeomanui.ts:17]: All could be in one file.
import { errorMessage, infoMessage, warnMessage  } from "./images/vscode/Messages";

check isInBAS in ./images/vscode/Messages.ts to return the right image.

**Review by tomer-epstein — CHANGES_REQUESTED** [review]

File: backend/src/yeomanui.ts (hunk 1 above)
Comment (slavik-lvovsky) [reply backend/src/yeomanui.ts]: done

File: backend/src/yeomanui.ts [hunk 2]
```diff
@@ -5,10 +5,12 @@ import * as _ from "lodash";
 import * as Environment from "yeoman-environment";
 import * as inquirer from "inquirer";
 import { ReplayUtils, ReplayState } from "./replayUtils";
+import * as vscode from 'vscode';
```
[the remark this answers is not in the document]
Comment (slavik-lvovsky) [reply backend/src/yeomanui.ts]: done

File: backend/src/yeomanui.ts [hunk 3]
```diff
@@ -5,10 +5,12 @@ import * as _ from "lodash";
...
 const datauri = require("datauri"); // eslint-disable-line @typescript-eslint/no-var-requires
 const titleize = require('titleize'); // eslint-disable-line @typescript-eslint/no-var-requires
 const humanizeString = require('humanize-string'); // eslint-disable-line @typescript-eslint/no-var-requires
-import * as defaultImage from "./defaultImage";
+import * as defaultImage from "./images/defaultImage";
+import {infoVSCode, infoTheia, warnVSCode, warnTheia, errorVSCodeDark, errorTheiaDark, errorVSCodeLight, errorTheiaLight} from "./images/imageMessage"
```
[the remark this answers is not in the document]
Comment (slavik-lvovsky) [reply backend/src/yeomanui.ts]: done

**Comment by slavik-lvovsky** [comment]
Approved

File: backend/src/images/messageImages.ts [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
...2xpcFBhdGg+DQogIDwvZGVmcz4NCiAgPGcgaWQ9ImVycm9yX3RoZWlhX2xpZ2h0IiBjbGlwLXBhdGg9InVybCgjY2xpcC1lcnJvcl90aGVpYV9saWdodCkiPg0KICAgIDxnIGlkPSJHcm91cF8zNTMxIiBkYXRhLW5hbWU9Ikdyb3VwIDM1MzEiIHRyYW5zZm9ybT0idHJhbnNsYXRlKC0xIC0xLjAwNCkiPg0KICAgICAgPHBhdGggaWQ9IlBhdGhfMTYwOCIgZGF0YS1uYW1lPSJQYXRoIDE2MDgiIGQ9Ik0yMzAuMzUxLTEwMi40YTguNSw4LjUsMCwwLDAtOC41LDguNSw4LjUsOC41LDAsMCwwLDguNSw4LjUsOC41LDguNSwwLDAsMCw4LjUtOC41LDguNSw4LjUsMCwwLDAtOC41LTguNVptMy41NTQsMTIuMzc2YS43OTMuNzkzLDAsMCwxLS41NTkuMjI3LjguOCwwLDAsMS0uNTY3LS4yMzdsLTIuNDQtMi40NzMtMi41NTgsMi42MjdhLjc5My43OTMsMCwwLDEtLjU2Ny4yNDEuNzkzLjc5MywwLDAsMS0uNTU2LS4yMjcuNzkzLjc5MywwLDAsMS0uMjQzLS41NjEuNzkzLjc5MywwLDAsMSwuMjI4LS41NjdsMi41ODItMi42NS0yLjM2NS0yLjRhLjc5My43OTMsMCwwLDEtLjIzMS0uNTY1Ljc5My43OTMsMCwwLDEsLjIzOS0uNTYyLjc5My43OTMsMCwwLDEsLjU2NS0uMjMxLjc5My43OTMsMCwwLDEsLjU2Mi4yMzlsMi4zNDMsMi4zNzMsMi4zMDktMi4zNzFhLjc5NC43OTQsMCwwLDEsLjU2LS4yNDIuNzkzLjc5MywwLDAsMSwuNTY2LjIyNy43OTMuNzkzLDAsMCwxLC4yNDMuNTYxLjc5My43OTMsMCwwLDEtLjIyOC41NjdsLTIuMzMxLDIuMzkyLDIuNDYyLDIuNDkzYS43OTMuNzkzLDAsMCwxLC4yMjkuNTY3Ljc5My43OTMsMCwwLDEtLjI0My41NjJaIiB0cmFuc2Zvcm09InRyYW5zbGF0ZSgtMjIwLjg1MSAxMDMuNDA0KSIgZmlsbD0iI2U4NGQ0ZCIvPg0KICAgIDwvZz4NCiAgPC9nPg0KPC9zdmc+DQo="
```
Comment (tomer-epstein) [inline backend/src/images/messageImages.ts:17]: errorTheiaLight --> not used.

**Pull request merged**